api/db-cdr/changelog-schema/db.changelog-1.xml

```diff
@@ -42,13 +42,10 @@
         <constraints nullable="true"/>
       </column>
     </createTable>
-    <createIndex
-            indexName="idx_type_parent_id"
-            tableName="criteria"
-            unique="false">
-      <column name="type"/>
-      <column name="parent_id"/>
-    </createIndex>
+
+    <modifySql dbms="mysql">
+      <append value="ENGINE=INNODB DEFAULT CHARSET=UTF8"/>
```

Comment:
Why is this needed? I thought our DB setup script did this for us....

Reply:
i guess i'm not following. What db setup script does this?

Reply:
UTF8 is set here:

https://github.com/all-of-us/workbench/blob/master/api/db-cdr/create_db.sql#L1

InnoDB is the default engine for MySQL so I'm not sure you need to specify that. 

Did things break if you didn't include this?

Reply:
I'm not sure... I was just following an example for full text search. I can remove it and see what happens.

Reply:
Still works after removing this modifySql section.